test/unit/double_free.c

```diff
@@ -101,10 +103,41 @@ TEST_BEGIN(test_small_double_free_tcache) {
 }
 TEST_END
 
+TEST_BEGIN(test_small_double_free_arena) {
+	test_skip_if(!config_debug);
+	test_skip_if(opt_debug_double_free_max_scan == 0);
```

Comment:
It looks we don't have to depend on the scan check for double frees?

Reply:
Yes, removed.